code/src/components/Form.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { quests } from 'reducer/quests'
+
+export const Form = () => {
+  const dispatch = useDispatch()
+
+  const categoriesList = useSelector((state) => state.quests.categories)
+  const [text, setText] = useState()
+  const [category, setCategory] = useState(categoriesList[0])
+
+  const handleChangeText = (e) => { setText(e.target.value) }
+  const handleChangeCategory = (e) => setCategory(e.target.value)
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    dispatch(quests.actions.createQuest({ text, category }))
+    setText('')
+  }
+  const handleDeleteAll = () => (dispatch(quests.actions.deleteAllQuests()))
+
+  return (
+    <>
```

Comment:
No need for a fragment here. You only need fragments when you want to return multiple elements, but here there's just one `<form>` so you can return it directly.